src/bundle.c

```diff
@@ -1108,12 +1108,30 @@ static int lex_sort(const void *a, const void *b)
  */
 int list_local_bundles()
 {
+	char *name;
 	char *path = NULL;
 	DIR *dir;
 	struct dirent *ent;
 	struct list *bundles = NULL;
 	struct list *item = NULL;
+	struct manifest *MoM = NULL;
+	struct file *bundle_manifest = NULL;
+	int current_version;
+	bool mix_exists;
+
+	current_version = get_current_version(path_prefix);
+	if (current_version < 0) {
+		goto skip_mom;
+	}
+
+	mix_exists = (check_mix_exists() & system_on_mix());
+	MoM = load_mom(current_version, false, mix_exists, NULL);
+	if (!MoM) {
+		printf("Warning: Could not determine which installed bundles are experimental\n");
+		goto skip_mom;
```

Comment:
you don't need this goto

Reply:
true, the code look different before and it needed it, but at some point it changed and forgot to remove it. Thanks for noticing it.